### PR TITLE
Add Python 3.9 image with python-oracledb

### PR DESCRIPTION
**Pull request by cjbj**
Everyone uses Oracle Database, right?

### CODE REVIEW

File: OracleLinuxDevelopers/oraclelinux8/python/3.9-oracledb/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+FROM ghcr.io/oracle/oraclelinux:8
+
+RUN dnf -y module disable python36 && \
+    dnf -y module enable python39 && \
+    dnf -y install python39 python39-pip python39-setuptools python39-wheel && \
+    dnf -y install oraclelinux-developer-release-el8 && \
+    dnf -y install python39-oracledb && \
```
Comment (Djelibeybi) [inline OracleLinuxDevelopers/oraclelinux8/python/3.9-oracledb/Dockerfile:10]:
```suggestion
FROM ghcr.io/oracle/oraclelinux8-python:3.9

RUN dnf -y install oraclelinux-developer-release-el8 && \
    dnf -y install python39-oracledb && \
```

File: OracleLinuxDevelopers/oraclelinux8/python/3.9-oracledb/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    # Optionally install Oracle Instant Client to use python-oracledb Thick mode
+    # dnf -y install oracle-instantclient-release-23ai-el8 && \
+    # dnf -y install oracle-instantclient-basic && \
+    rm -rf /var/cache/dnf
+
+CMD ["/bin/python3", "--version"]
```
Comment (Djelibeybi) [inline OracleLinuxDevelopers/oraclelinux8/python/3.9-oracledb/Dockerfile:16]:
```suggestion

```

**Review by Djelibeybi — CHANGES_REQUESTED** [review]
My suggested changes essentially switch to using our existing Python 3.9 image instead. Let me know if there's a particular reason you don't want to do this.

File: OracleLinuxDevelopers/oraclelinux8/python/3.9-oracledb/Dockerfile (hunk 2 above)
Comment (cjbj) [reply OracleLinuxDevelopers/oraclelinux8/python/3.9-oracledb/Dockerfile:16]: Can do.

I had just based it on https://github.com/oracle/docker-images/blob/main/OracleLinuxDevelopers/oraclelinux8/python/3.6-oracledb/Dockerfile

**Review by Djelibeybi — APPROVED** [review]
It's probably worth changing the Python 3.6 one as well, but it's not urgent.

**Pull request merged**